.github/actions/install/action.yml

```
@@ -12,13 +12,13 @@ runs:
     - name: Install Go
       uses: actions/setup-go@v5
       with:
-        go-version: "1.22.x"
+        go-version: "1.23.x"
```

We could get this value automatically from the go.mod ...

```suggestion
        go-version-file: "providergo.mod"
```

---

Oh nice, didn't know that!